goon/browserassets/css/browserOutput.css

```diff
@@ -379,6 +379,7 @@ h1.alert, h2.alert			{color: #000000;}
 .orangei				{color: #ffa500; font-style: italic;}
 .orangeb				{color: #ffa500; font-weight: bold;}
 .resonate				{color: #298F85;}
+.healthscan_oxy			{color: #007ECC;}
```

Comment:
Something like would be better (heres the website btw https://webaim.org/resources/contrastchecker/)
```suggestion
.healthscan_oxy			{color: #0074BD;}
```
![image](https://github.com/ParadiseSS13/Paradise/assets/91113370/48c6d9ae-c00c-42e9-8200-3eccdff1a601)